predict.py

```diff
@@ -324,6 +318,10 @@ def base_predict(
         torch_device = torch.device("cuda")
         init_image = None
 
+        if not seed:
```

Comment:
hmm.  fp8 predict does come up with a seed, but there's value in logging the seed s.t. the user can see it for subsequent predictions.

Reply:
ah jk it's there isn't it. fair enough, this'll work.